docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
 
-    - ```python
-      message = 'i wanted to watch movie'
-      entity_name = 'movie'
-      structured_value = 'inferno'
-      fallback_value = None
-      bot_message = None
-      ```
+   For example, if user says *"Nearby ATMs"*. In this example user has not provided any information about his location in the chat but, we can pass a *fallback_value* that will contain its location that can be obtained from its profile or third party apis (like geopy, etc).
 
-    - *Python:* 
+5. **source_language**: It is the language in which user message is passed. We have to pass ISO 639-1 code of the language here.
 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_text
-      output = get_text(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
+6. **bot_message**: previous message from a bot/agent. This is an important parameter, many times the entity value relies on the message from the bot/agent i.e. what is bot saying? or asking?
 
-    - *CURL command:*
+   For example, bot might ask for departure date to book a flight and user might reply with a date. Now, it is difficult to disambiguate whether its departure date or arrival date unless, we know what bot is asking for?
 
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
+   ```
+   bot: Please help me with date of departure?
+   user: 23rd March
+   ```
 
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20wanted%20to%20watch%20movie&entity_name=movie&structured_value=inferno&fallback_value=None&bot_message='
-      ```
+### **Output Format**
 
-    - *CURL Output*:
+The output of entities detected will be stored in a list of dictionary containing the following structure:
 
-    - ​
+```
+[
+    {
+        "entity_value": entity_value,
+        "detection": detection_method,
+        "original_text": original_text
+    },
+]
+```
 
-      ```json
-      {
-        "data": [
-          {
-            "detection": "structure_value_verified",
-            "original_text": "inferno",
-            "entity_value": {
-              "value": "Inferno"
-            }
-          }
-        ]
-      }
-      ```
+Consider the following example for detailed explanation:
 
-  - ​
+```
+"I want to order two litre pepsi."
+```
 
+- entity_value: This will store the value of entity (i.e entity value) that is detected. The output will be in dictionary format. For example, `{"value": "2", "unit": "quantity"}`.
+- detection: This will store how the entity is detected i.e. whether from *message*, *structured value* or *fallback value*.
+- original_text: This will store the actual value that is detected. For example, `two`.
 
-### phone_number
+```json
+Example 1:
+input:  message = 'I want to order 2 burgers from mainland china at 3 pm '
```

Comment:
What is the purpose of this example ? Do we want people to be able to copy it easily into Python shell ? Then enter a new line after the "input: " And remove the  indent

Reply:
Same for other examples in the file